types/tizen-common-web/index.d.ts

```diff
@@ -1,84 +1,14699 @@
+     *
+     * @privilegeLevel public
+     * @privilegeName http://tizen.org/privilege/filesystem.write
+     *
+     * @param file File to open.
+     * @param mode File mode for the opened archive. Determines which operations are available.
+     * @param onsuccess Callback method to be invoked when archive is opened successfully.
+     * @param onerror Callback method to be invoked when an error occurs.
+     * @param options Additional options for initializing the ArchiveFile instance.
+     *
+     * @returns Task ID which can be used to cancel the operation with abort().
+     *
+     * @throw WebAPIException with error type TypeMismatchError, if parameter type does not match.
+     * @throw WebAPIException with error type SecurityError, if the application does not have the privilege to call this method or the application does not have privilege to access the storage. For more information, see [Storage privileges](#StorageRemark).
+     */
+    open(file: FileReference, mode: FileMode, onsuccess: ArchiveFileSuccessCallback, onerror?: ErrorCallback | null, options?: ArchiveFileOptions | null): number;
+}
+/**
+ * This interface represents a set of filters.
+ *
+ * It represents the query statement for the specified value of _matchValue_ by the rule of _matchFlag_.
+ *
+ * If no _matchValue_ is defined, the filter matches all objects that have the attribute
+ * defined (same as the "EXISTS" filter works), otherwise, it only matches objects which have an attribute that match
+ * the specified value.
+ */
+export class AttributeFilter extends AbstractFilter {
+    constructor(attributeName: string, matchFlag?: FilterMatchFlag | null, matchValue?: any);
+    /**
+     * The name of the object attribute used for filtering.
+     *
+     * This is the name of the object attribute exactly as it is defined in
+     * the object's interface. For attributes of complex type, use fully-qualified names
+     * (such as "geolocation.latitude" to filter a video or image content's latitude in a geolocation).
+     *
+     * For attributes of an array type, the filter will match if any value in the array
+     * matches.
+     */
+    attributeName: string;
+    /**
+     * The match flag used for attribute-based filtering.
+     *
+     * By default, this attribute is set to "EXACTLY".
+     */
+    matchFlag: FilterMatchFlag;
+    /**
+     * The value used for matching.
+     *
+     * The filter will match if the attribute value matches the given matchValue.
+     *
+     * This value is not used if the _matchFlag_ is set to "EXISTS".
+     * By default, this attribute is set to null.
+     */
+    matchValue: any;
+}
+export interface AttributeFilter {
```

Comment:
Another case of incorrectly declaring the *static* (i.e., constructor) side of the class on an interface that represents the *instance*. Please remove.